src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -91,7 +92,11 @@ public Task RunAsync()
                 }
                 catch (Exception ex)
                 {
-                    ColorConsole.WriteLine($"{DateTime.Now} Failed to get leaderboard. ".Red(), ex.Message.DarkRed());
+                    ColorConsole.WriteLine(
+                        $"{DateTime.Now} ".DarkGray(),
+                        "Failed to get leaderboard. ".Red(),
+                        ex is ServerException ? "Is the web API down?".Yellow() : ex.Message.DarkRed());

```

Comment:
Would it be better to state something like "Connection to web API lost."? It looks weird to have an error message ask a question, especially if it will be on monitors throughout a convention.

Reply:
I did that deliberately to serve as an instant reminder to whoever is running the HashBus for what needs to be checked. The other (non-question) phrase I was considering was:

> The web API appears to be down.

I prefer that to "connection" since there is no persistent connection.

Thoughts?

Reply:
Makes sense to me 👍

Reply:
OK, I'll amend the first commit and force push.